mycroft/stt/deepspeech_stt.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import absolute_import, division
+
+from mycroft.stt import STT
+from mycroft.util.log import LOG
+from timeit import default_timer as timer
+from os.path import exists, join, dirname
+from mycroft.util.download import download
+import tarfile
+from time import sleep
+
+
+class DeepSpeechSTT(STT):
+    URL = "https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/" \
```

Comment:
how big is this model? are we sure that existing mark1s and picrofts can download this without accidentally filling disk?

Reply:
2.1 GB